fake-news-detection/src/fake_news_detection/pipelines/data_processing/pipeline.py

```
@@ -0,0 +1,136 @@
+from kedro.pipeline import Pipeline, node, pipeline
+from kedro.pipeline.modular_pipeline import pipeline
```

Ruff (0.11.9)

Redefinition of unused `pipeline` from line 1

(F811)